service/src/main/resources/templates/unstructured_observation_statement_template.mustache

```diff
@@ -16,5 +16,8 @@
         {{{referenceRange}}}
         {{{code}}}
         {{{interpretation}}}
+        {{#agentReference}}
```

Comment:
When is this set? I can't see any reference to this in the `ObservationStatementMapper`

Reply:
Changed to participant